feat(react-scheduler): add Vertical Grouping (#2740)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/reference/grouping-panel.md

rowComponent | ComponentType&lt;[GroupingPanel.RowProps](#groupingpanelrowprops)&gt; |  A component that renders a row on the grouping panel.
cellComponent | ComponentType&lt;[GroupingPanel.CellProps](#groupingpanelcellprops)&gt; | A component that renders a cell in a row on the grouping panel.

### GroupingPanel.VerticalLayoutProps

> Describes properties passed to a component that renders the grouping panel vertically.

Reviewer: ```suggestion
Describes properties passed to a component, which renders the grouping panel vertically.
```

rowComponent | ComponentType&lt;[GroupingPanel.RowProps](#groupingpanelrowprops)&gt; |  A component that renders a row on the grouping panel.
cellComponent | ComponentType&lt;[GroupingPanel.CellProps](#groupingpanelcellprops)&gt; | A component that renders a cell in a row on the grouping panel.

### GroupingPanel.VerticalLayoutProps

Describes properties passed to a component that renders the grouping panel vertically.

Field | Type | Description
------|------|------------
groups | Array&lt;Array&lt;[Group](#group)&gt;&gt; | Specifies the final representation of Scheduler's groups and the order they will be rendered in.

> rowSpan | number | Indicates the number of rows in the Scheduler's timetable.

Reviewer 2: ```suggestion
rowSpan | number | The number of rows in the timetable.
```

> viewType | string | Specifies the view the layout is rendered in.

Reviewer 2: ```suggestion
viewType | string | The view in which the layout is rendered.
```

------|------|------------
group | [Group](#group) | The group the cell represents.
colSpan | number | The number of columns the cell spans.
rowSpan? | number | The number of rows the cell spans.
left | number | The cell's offset from the left.
height? | number | The cell's height.

> topOffSet? | number | Specifies the distance between the cell's text and the top edge of the Scheduler.

Reviewer 2: ```suggestion
topOffSet? | number | The distance between the cell's text and the top edge of the Scheduler.
```

> textStyle? | object | Specifies the CSS properties to apply to the Cell's text.

Reviewer 2: ```suggestion
textStyle? | object | CSS properties to be applied to the cell's text.
```
